src/components/paradox/results/personality/career/inline-jobs/job.js

```diff
@@ -0,0 +1,67 @@
+import PropTypes from "prop-types";
+import {faLocationDot} from "@fortawesome/free-solid-svg-icons";
+import Icon from "lib/helpers/icon";
+import DangerousHTML from "lib/helpers/dangerous-html";
+import style from "./style.scss";
+
+function InlineJob({job, jobSource = "Indeed", translate}) {
```

Comment:
This shouldn't need a default since it's already set by defaultProps

Reply:
Good catch! Redundant default removed!